src/Generator/AttributeGenerator/AttributePart.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator\AttributeGenerator;
+
+//TODO Enum in PHP8.1
+final class AttributePart
```

Comment:
Let's mark all internally used symbols as `@internal`, to allow refactoring/rewrite as soon as we are allowed to.